databroker/broker.py

```diff
@@ -170,6 +172,56 @@ def __iter__(self):
                         break
 
 
+CONFIG_SEARCH_PATH = (os.path.expanduser('~/.config/databroker/'),
```

Comment:
I think a clean way to do this is :

```python
import sys
from pathlib import Path
local_etc = Path(sys.executable).parents[1] / 'etc' / 'databroker'
```

which will work without activation and with venvs.